lib/language_pack/rails4.rb

```diff
@@ -95,12 +95,15 @@ def run_assets_precompile_rake_task
           log "assets_precompile", :status => "success"
           puts "Asset precompilation completed (#{"%.2f" % precompile.time}s)"
 
-          puts "Cleaning assets"
-          rake.task("assets:clean").invoke(env: rake_env)
+          clean_task = rake.task("assets:clean")
+          if clean_task.not_defined?
```

Comment:
if the clean task is not defined we run it?

Reply:
Right, also the test was wrong. Fixed both.